pkgs/by-name/wo/world-wall-clock/package.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "world-wall-clock";
+  version = "0.1.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ddelabru";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-OTBYSStCFBrZ8JutrhyyJpi7vRvBAUK0EKTtjvbi13I=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
```

```suggestion
  build-system = with python3.pkgs; [
```
And `dependencies` for Python dependencies.

---

Changed `nativeBuildInputs` for `build-system` and `propagatedBuildInputs` for `dependencies`.